packages/stg-evm-v2/hardhat.config.ts

```diff
@@ -162,6 +162,13 @@ const networks: NetworksUserConfig = {
     //
     // Mainnet
     //
+    'abstract-mainnet': {
+        eid: EndpointId.ABSTRACT_V2_MAINNET,
+        url: process.env.RPC_URL_ABSTRACT_MAINNET,
+        accounts: mainnetAccounts,
+        safeConfig: getSafeConfig(EndpointId.ABSTRACT_V2_MAINNET),
+        timeout: DEFAULT_NETWORK_TIMEOUT,
+    },
```

Comment:
This was the config in my `hardhat.config.ts`

```
    'abstract-mainnet': {
        eid: EndpointId.ABSTRACT_V2_MAINNET,
        url: process.env.RPC_URL_ABSTRACT_MAINNET,
        accounts: mainnetAccounts,
        safeConfig: getSafeConfig(EndpointId.ABSTRACT_V2_MAINNET),
        timeout: DEFAULT_NETWORK_TIMEOUT,
        zksync: true,
        ethNetwork: 'ethereum-mainnet',
    }
```

Please add the last 2 properties!

If you run into issues during deployment/configuration, try adding the following as well:




```        
gasMultiplier: 2.0,
```

If you _still_ run into issues, try adding (keeping the gasMultiplier from above too):

```
 gas: 10000000,
```

Reply:
Thank you, added!